docs/jwst/pipeline/calwebb_detector1.rst

```diff
@@ -15,54 +15,54 @@ are in the form of one or more ramps (integrations) containing accumulating
 counts from the non-destructive detector readouts and the output is a corrected
 countrate (slope) image.
 
-There are two unique configuration files to be used to control this pipeline,
-depending on whether the data are to be treated as a Time Series Observation (TSO).
-Non-TSO exposures use the ``calwebb_detector1`` configuration, which applies all
-applicable steps to the data. The ``calwebb_tso1`` configuration, on the other hand,
-should be used for TSO exposures, for which some steps are set to be skipped by
-default (see the list of steps in the table below). Both configurations call the
-``Detector1Pipeline``; the only difference is which steps are applied.
+There are two unique configurations used to control this pipeline,
+depending on whether the data are to be treated as a Time Series Observation
+(TSO). The configuration is controlled by CRDS and the reftype
```

Comment:
Agreed with the wording change.

Question though, and this applies to a number of other cases in the documentation: Information such as this is beginning to cross the line of "what is and how does one use the software" to "use-case scenarios and science-oriented discussion of how the software is utilized". Granted it is a fuzzy line, and this may be the only place such information is currently discussed, but such information feels like it should be in some sort of user-handbook about the specific instruments and observation modes and their calibration. Oh wait, such a thing exists: JDocs! 

With the move to a more data-driven process, this type of documentation really doesn't belong here. Of course, this is a much bigger re-org that will not be solved by this PR, but should be considered in future revisions.